infrastructure/tooling/src/smoketest/checks/role.js

```diff
@@ -27,6 +27,18 @@ exports.tasks.push({
         'project:taskcluster:smoketest:abc/*' ],
     };
     assert.deepEqual(expandedRole.scopes, expectedScopes.scopes);
-    await auth.deleteRole(roleId);
+    const query = {};
+    const anHourAgo = Date.now() - (1000*60*60);
+    while (1) {
+      const res = await auth.listRoles2();
+      if(res.roles[0].roleId.includes('project:taskcluster:smoketest:') === true && res.roles[0].lastModified < anHourAgo){
```

Comment:
There's no need to compare a value to true -- booleans are booleans :)

Reply:
Yes, exactly :)

Reply:
Can you please take a look now to my changes ? :)

Reply:
Can you please take a look at my changes please :)
@djmitche